# Check XL-320 and XL-430 reference doc for
# the complete register table

int8 motor_type # 1 (Stepper) 2 (XL-430) or 3 (XL-320) or 4 (XL-330) or 5 (XC-430)
uint8 id
int32 value
int32 reg_address
int32 byte_number
---
int32 status
string message
